### ISSUE TITLE
Calls to Get-AzureSqlDatabaseUsages fail each time

### ISSUE THREAD

**Issue by mandrie** [Azure/azure-powershell#480]
I was trying out the Get-AzureSqlDatabaseUsages call today and it fails each time for me with a cryptic error and I also cannot find any information online. I assume this is a newer cmdlet, here is the output (dbname and server name redacted):

```powershell
PS C:\Users\mandrie\Documents\WindowsPowerShell> Get-AzureSqlDatabaseUsages -ServerName foo -DatabaseName bar
Get-AzureSqlDatabaseUsages : Data at the root level is invalid. Line 1, position 1.
At line:1 char:1
+ Get-AzureSqlDatabaseUsages -ServerName foob -DatabaseName bar...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureSqlDatabaseUsages], XmlException
    + FullyQualifiedErrorId : Microsoft.WindowsAzure.Commands.SqlDatabase.Database.Cmdlet.GetAzureSqlDatabaseUsages
```

```powershell
PS C:\Users\mandrie\Documents\WindowsPowerShell> gcm Get-AzureSqlDatabaseUsages

CommandType     Name                                               Version    Source                                                                                                                                               
-----------     ----                                               -------    ------                                                                                                                                               
Cmdlet          Get-AzureSqlDatabaseUsages                         0.9.2      Azure   
```

From the full debug log I can see that it returns what I want to get (size of the database) but it fails before actually giving back the object:

```powershell
PS C:\Users\mandrie\Documents\WindowsPowerShell> Get-AzureSqlDatabaseUsages -ServerName foo -DatabaseName bar -Verbose -Debug
...
DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
GET

Absolute Uri:
https://management.core.windows.net/REDACTED/services/sqlservers/servers/foo/databases/bar/usages

Headers:
x-ms-version                  : 2012-03-01

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
x-ms-servedbyregion           : REDACTED
X-Content-Type-Options        : nosniff
x-ms-request-id               : REDACTED
Cache-Control                 : no-store,no-cache
Date                          : Tue, 09 Jun 2015 20:05:18 GMT
Server                        : REDACTED,(rd_rdfe_stable.150604-1401),Microsoft-HTTPAPI/2.0

Body:
{
  "Value": [
    {
      "CurrentValue": 2444427264,
      "DisplayName": "Database Size",
      "Limit": 268435456000,
      "Name": "database_size",
      "ResourceName": "foobar",
      "Unit": "Bytes"
    }
  ]
}


Get-AzureSqlDatabaseUsages : Data at the root level is invalid. Line 1, position 1.
At line:1 char:1
+ Get-AzureSqlDatabaseUsages -ServerName foo-DatabaseName bar...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureSqlDatabaseUsages], XmlException
    + FullyQualifiedErrorId : Microsoft.WindowsAzure.Commands.SqlDatabase.Database.Cmdlet.GetAzureSqlDatabaseUsages
 
DEBUG: 1:05:20 PM - GetAzureSqlDatabaseUsages end processing.
<!---@tfsbridge:{"tfsId":4151386}-->

**Comment by akromm** [comment]
Are you able to reproduce this using v0.9.4 of the PowerShell cmdlets?

**Comment by mandrie** [comment]
yep, just updated to .9.4 and I still get the same error:
````powershell
PS C:\Users\mandrie\Documents\WindowsPowerShell> Get-AzureSqlDatabaseUsages -ServerName foo -DatabaseName bar
Get-AzureSqlDatabaseUsages : Data at the root level is invalid. Line 1, position 1.
At line:1 char:1
+ Get-AzureSqlDatabaseUsages -ServerName foo -DatabaseName bar ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureSqlDatabaseUsages], XmlException
    + FullyQualifiedErrorId : Microsoft.WindowsAzure.Commands.SqlDatabase.Database.Cmdlet.GetAzureSqlDatabaseUsages
 

PS C:\Users\mandrie\Documents\WindowsPowerShell> gcm Get-AzureSqlDatabaseUsages

CommandType     Name                                               Version    Source                                                                                                                                               
-----------     ----                                               -------    ------                                                                                                                                               
Cmdlet          Get-AzureSqlDatabaseUsages                         0.9.4      Azure 
````

**Comment by snarasingh** [comment]
same issue exists even if we use 0.9.5

**Comment by devigned** [comment]
@akromm any thoughts on this one?

**Comment by akromm** [comment]
I will look into how to fix this.

**Comment by mandrie** [comment]
Any ETA on when this will be fixed? It still fails the same in version 9.8. Please let me know if you need an additional repro

Thanks!
Mark

**Comment by mandrie** [comment]
Hi @akromm, just curious if this was already fixed? I can no longer repro it on version 1.0.2, so I assume we are good to close this issue

**Comment by akromm** [comment]
Yes, this has been resolved.  

I apologize for not responding sooner.
